cnf-tests/testsuites/e2esuite/bond/bond_sriov.go

```diff
@@ -71,24 +71,23 @@ var _ = Describe("[sriov] Bond CNI integration", func() {
 			podDefinition := pods.DefineWithNetworks(namespaces.BondTestNamespace, []string{
 				fmt.Sprintf("%s/%s", namespaces.SRIOVOperator, "test-network"),
 				fmt.Sprintf("%s/%s", namespaces.SRIOVOperator, "test-network"),
-				fmt.Sprintf("%s/%s@%s", namespaces.BondTestNamespace, "bond", bondLinkName),
+				fmt.Sprintf("%s/%s@%s", namespaces.BondTestNamespace, "bondifc", bondLinkName),
```

Comment:
please put the network-attach-name in a variable like the bondLinkName this will make our life easier in the future